bool doPostProcessing
---
sensor_msgs/PointCloud2 spatialMap
